compiler/AST/CForLoop.cpp

```diff
@@ -151,9 +149,6 @@ CForLoop* CForLoop::copy(SymbolMap* mapRef, bool internal)
   for_alist(expr, body)
     retval->insertAtTail(expr->copy(map, true));
 
-  if (internal == false)
-    update_symbols(retval, map);
-
```

Comment:
The PR message should say something about these copy changes. The change is migrating LoopStmts to use the usual strategy of overriding copyInner rather than overriding copy itself.